<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PipeLine Creator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>PipesTransparent.png</normaloff>PipesTransparent.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 254, 255);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(158, 255, 131);
border-top-color: rgb(35, 255, 182);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>PipesTransparent.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AbrirProcesso">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 199, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Calcular Processos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConectarSQL">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 199, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Connectar SQL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CarregarPIPE">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 199, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Carregar .PIPE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Permanent Marker</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 172, 6);</string>
    </property>
    <property name="text">
     <string>PIPE CREATOR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="infodev"/>
    <addaction name="cred"/>
   </widget>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFerramentas">
   <property name="text">
    <string>Ferramentas</string>
   </property>
  </action>
  <action name="actionConnectar_SQL">
   <property name="text">
    <string>Connectar SQL</string>
   </property>
  </action>
  <action name="actionCarregar_PIPE">
   <property name="text">
    <string>Carregar .PIPE</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="infodev">
   <property name="text">
    <string>Informações do Desenvolvedor</string>
   </property>
  </action>
  <action name="cred">
   <property name="text">
    <string>Créditos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
